inputPageCode.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once('session.php');
+require_once('functions.php');
+
+$db = linkDB();
+$mice = getDataFromDB($db);
+
+if (!empty($_POST['mouseName']) && !empty($_POST['mouseBrand']) && !empty($_POST['mouseWeight']) && !empty($_POST['wirelessInput'])) {
+    $mouseName = $_POST['mouseName'];
+    $mouseBrand = $_POST['mouseBrand'];
+    $mouseWeight = $_POST['mouseWeight'];
+    if ($_POST['wirelessInput'] == 'yes') {
```

Comment:
if you change the input values this can just be `if ($_POST['wirelessinput']) {`

Reply:
infact, if you change the values you wont need this at all, you can just typecast it into an int